source/common/network/io_socket_handle_impl.h

```diff
@@ -2,33 +2,28 @@
 
 #include "envoy/network/io_handle.h"
 
-#include "common/common/assert.h"
-
 namespace Envoy {
 namespace Network {
 
 /**
  * IoHandle derivative for sockets
  */
-class IoSocketHandle : public IoHandle {
+class IoSocketHandleImpl : public IoHandle {
```

Comment:
Worth commenting somewhere that the fd will close on destruct, especially as it's a light-weight enough class someone might create a local-scope instance.

Reply:
Commented in destructor